tests/e2e/cucumber/features/smoke/urlJourneys.feature

```diff
@@ -0,0 +1,27 @@
+Feature: web can be navigated through urls
+
+
+  Scenario: navigate web directly through urls
+    Given "Admin" creates following user using API
+      | id    |
+      | Alice |
+      | Brian |
+    And "Alice" logs in
+    And "Alice" creates the following folders in personal space using API
+      | name   |
+      | FOLDER |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile | content      |
+      | lorem.txt  | some content |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile | content     |
+      | lorem.txt  | new content |
+    When "Alice" navigates to "versions" details panel of file "lorem.txt" through the URL
+    And "Alice" restores following resources
+      | resource  | to | version | isDirectUrlNavigation |
+      | lorem.txt | /  | 1       | true                  |
```

Comment:
maybe `openDetailsPanel`?
```suggestion
      | resource  | to | version | openDetailsPanel |
      | lorem.txt | /  | 1       | false            |
```

Reply:
Would it be a little misleading or confusing for someone adding new tests because we are opening the details panel but just not through the normal web sequence?

Reply:
I just thought that using `isDirectUrlNavigation` is tied to the navigation step above it. So, if we could make it general then maybe we can pair this step with other journeys as well.